rcl_yaml_param_parser/Doxyfile

```diff
@@ -0,0 +1,28 @@
+# All settings not listed here will use the Doxygen default values.
+
+PROJECT_NAME           = "rcl_yaml_param_parser"
+PROJECT_NUMBER         = master
+PROJECT_BRIEF          = "Parse a YAML parameter file and populate the C data structure."
+
+INPUT                  = ./include README.md
+USE_MDFILE_AS_MAINPAGE = README.md
+RECURSIVE              = YES
+OUTPUT_DIRECTORY       = doc_output
+
+EXTRACT_ALL            = YES
+SORT_MEMBER_DOCS       = NO
+
+GENERATE_LATEX         = NO
+
+ENABLE_PREPROCESSING   = YES
+MACRO_EXPANSION        = YES
+EXPAND_ONLY_PREDEF     = YES
+PREDEFINED             += RCL_PUBLIC=
```

Comment:
This one uses RCL_YAML_PARAM_PARSER_PUBLIC and I'm not sure RCL_WARN_UNUSED is actually used in this package.

Reply:
https://github.com/ros2/rcl/pull/634/commits/54a142738f54bbd17c7b41a5f89eef40fd68595f